packages/grafana-e2e/cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```

Comment:
I assume the code commented out is on purpose?

Reply:
Cypress internal example stuff, I'll remove it moving forward.